README.md

```diff
@@ -1,2 +1,33 @@
-# Darko-Online-Class
-This is a microverse first module ending project.
+# Darko Online Class
```

Comment:
- [ ] Kindly add a screenshot of your project here too and your Readme will look perfect and professional.